Config.xcconfig

```diff
@@ -1,5 +1,5 @@
 APP_DISPLAY_NAME = iAPS
-APP_VERSION = 2.3.2
+APP_VERSION = 2.3.3
```

Comment:
This PR adds new, non-breaking changes. Subsequently it should be a minor release, not a patch-release, i.e. 2.x.0

Reply:
I can't, since this version then will be too close to dev version, even exceeding some more recent dev versions. The dev and main release versions will get in better sync later, starting from 3.0.0, but right now it is how it is...

Reply:
You are right, good thinking. Maybe an idea: Loop has odd numbers for dev and even numbers for release. If I am not mistaking then dev is currently 2.4.x and latest main is 2.3.2. 

How about we the Loop order, and make 2.6.0 the next minor release for main, which puts dev at 2.7.0. 

All following changes to dev are then patch versions, so 2.7.1, 2.7.2, etc. and once you say it is release ready for the next minor release it is, main is bumped to 2.8.0 and dev automatically jumps to 2.9.0

I know this would skip 2.5.x, but maybe it would work? What do you think 😊

Reply:
Yes, perhaps something like that. Skipping a number would indeed solve the current version confusion.